### PR TITLE
Add open-cluster-management website

### PR DESCRIPTION
**Pull request by qiujian16**

### CODE REVIEW

File: ci-operator/config/open-cluster-management/website/OWNERS [hunk 1]
```diff
@@ -0,0 +1,5 @@
+approvers:
+- qiujian16
+
+reviewers:
+- qiujian16
```
Comment (petr-muller) [inline ci-operator/config/open-cluster-management/website/OWNERS:5]: Please add more people. Who will review your PRs?

Comment (qiujian16) [reply ci-operator/config/open-cluster-management/website/OWNERS:5]: added

Comment (petr-muller) [reply ci-operator/config/open-cluster-management/website/OWNERS:5]: Not to this OWNERS file though :)

Comment (qiujian16) [reply ci-operator/config/open-cluster-management/website/OWNERS:6]: ah sorry, added to this file also.

**Review by petr-muller — APPROVED** [review]

**Pull request merged**